README.md

```diff
@@ -37,6 +38,16 @@ Meshery facilitates adopting, configuring, operating and managing performance of
 </p>
 <br />
 <p style="clear:both;">
+<h2><a href="https://github.com/service-mesh-patterns">Service Mesh Patterns</a></h2>
+<a href="https://meshery.io"><img src="https://github.com/service-mesh-patterns/service-mesh-patterns/blob/master/.github/readme/images/service-mesh-pattern.png"
+style="margin:10px;" width="125px" 
+alt="Meshery - Service Mesh Patterns" align="left" /></a>
+
+Service mesh patterns help you get the most out of Linkerd, Consul, App Mesh, Istio, Kuma, Open Service Mesh or <a href="https://layer5.io/service-mesh-landscape">any service mesh</a>, including those using Envoy and use of WebAssembly. Each pattern can be used as a template and is customizable.
```

Comment:
Add the hyperlink for text `Service Mesh Patterns` here as well. Also, the name will be capitalized as shown.

Reply:
Ok will do it asap